themes/default/content/blog/introducing-the-puluminaries/index.md

```diff
@@ -0,0 +1,103 @@
+
+In addition to being a Puluminary, Komal is also a Pulumni, having spent time working to make Pulumi a better tool for developers -  they were ever-present in making our Python SDK better and championing [Automation API](https://www.pulumi.com/docs/guides/automation-api/)!
+
+### Paul Hicks
+
+<img src="paul-hicks.png" width="200" height="200" alt="Paul Hicks"/>
+<a data-track="github" href="https://github.com/tenwit"><i class="fab fa-github"></i></a>
+
+Paul is one of the most prolific helpers in [Pulumi Community Slack](https://slack.pulumi.com/)! Chances are, if you see a great answer to a question in Slack, it came from Paul.
+
+### Ringo De Smet
+
+<img src="ringo-smet.png" width="200" height="200" alt="Ringo De Smet"/>
+<a data-track="twitter" href="https://twitter.com/ringods"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/ringods"><i class="fab fa-github"></i></a>
+
+Ringo is one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/) - a place to interact and collaborate on Pulumi-based libraries, projects, and educational materials. Ringo (along with fellow Puluminary Simen A. W. Olsen) have brought their vision for expanding the Pulumi community to life with the creating of this community-operated space! Ringo also created the [Pulumi Resource Type for Concourse](https://github.com/ringods/pulumi-resource), and is ever-present in every community he is a part of, not just Pulumi's!
+
+### Rizel Scarlett
+
+<img src="rizel-scarlett.png" width="200" height="200" alt="Rizel Scarlett"/>
+<a data-track="twitter" href="https://twitter.com/blackgirlbytes"><i class="fab fa-twitter"></i></a>&nbsp;<a data-track="github" href="https://github.com/blackgirlbytes"><i class="fab fa-github"></i></a>&nbsp;<a data-track="linkedin" href="https://www.linkedin.com/in/rizel-bobb-semple/"><i class="fab fa-linkedin"></i></a>
+
+Rizel brings her remarkable talent for guidance and teaching to the Pulumi community. She has participated in many Pulumi events, including [Cloud Engineering Summit](https://www.pulumi.com/resources/from-code-to-cloud/), and [appearing on the PulumiVision livestream](https://youtu.be/2uQEIYuJBZ4).
+
+### Simen A. W. Olsen
+
+<img src="simen-olsen.png" width="200" height="200" alt="Simen W Olsen"/>
+<a data-track="github" href="https://github.com/cobraz"><i class="fab fa-github"></i></a>
+
+Another one of the founding board members of [the Pulumiverse](https://www.pulumi.com/blog/2022-03-30-introducing-pulumiverse/), Simen has a great passion for technology communities, but especially for the Pulumi community. Simen was instrumental on the [Pulumi GitHub Action](https://github.com/pulumi/actions) rewrite - designing and architecing it in his free time.
```

Comment:
"Simen has a great passion for technology communities, but especially for the Pulumi community." - this is a bit nebulous and a filler. Anything else great we can call out to? If not, that is fine. 

architecting also has a typo

Reply:
The github action thing was the big one Paul Stack mentioned - for the communities bit it was more about how he had a lot of thoughts about how to construct the board for Pulumiverse.